scripts/custom_lint_checks.py

```diff
@@ -269,6 +269,14 @@ def visit_functiondef(self, node):
         self.check_functiondef_yields(node, node_doc)
 
     def check_functiondef_params(self, node, node_doc):
```

Comment:
Also, here and below please include the `Returns` and return type. I think these functions do return a boolean type.

Reply:
Are all these functions that have return written but nothing after that return a boolen value?

Reply:
Ah, sorry. I don't think we need returns here. You can leave that out.

Reply:
okay. Thanks.